include/internal/quic_rcidm.h

```diff
@@ -0,0 +1,184 @@
+/*
+* Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+*
+* Licensed under the Apache License 2.0 (the "License").  You may not use
+* this file except in compliance with the License.  You can obtain a copy
+* in the file LICENSE in the source distribution or at
+* https://www.openssl.org/source/license.html
+*/
+
+#ifndef OSSL_INTERNAL_QUIC_RCIDM_H
+# define OSSL_INTERNAL_QUIC_RCIDM_H
+# pragma once
+
+# include "internal/e_os.h"
+# include "internal/time.h"
+# include "internal/quic_types.h"
+# include "internal/quic_wire.h"
+
+# ifndef OPENSSL_NO_QUIC
+
+/*
+ * QUIC Remote Connection ID Manager
+ * =================================
+ *
+ * This manages connection IDs for the TX side, which  is to say that it tracks
+ * remote CIDs (RCIDs) which a peer has issued to us and which we can use as the
```

Comment:
The sentence should be split for better legibility. I.e. `This manages connection IDs for the TX side. It tracks remote CIDs (RCIDs) which a peer has issued to us and which we can use as the DCID of packets we transmit...`